src/api/api.types.tsx

```diff
@@ -56,3 +56,13 @@ export interface Unit extends UnitPost, CreatedModifiedMixin {
   id: string;
   code: string;
 }
+
+// ------------------------------------ Usage Status ------------------------------------------------
```

Comment:
I think either this and handlers should use all caps or otherwise all of them should be like this.

Reply:
I noticed this in the systems one and I have changed it there (#697) also it is inconsistent in the handlers so that need to be addressed in the audit

Reply:
As its such a small I would just do it there now if possible. Its possible we could miss these things if we don't make a note.